scripts/drone/steps/build.star

```diff
@@ -0,0 +1,11 @@
+def compile_build_cmd():
```

Comment:
As a first step I'd say we can include this definition in the `lib.star`, and decide if we need separate steps in separate `*.star` files.
Also we could rename it from `build.star`, to something like `build_cmd.star`, in order not to conflict with the `pipelines/build.star` and to be more descriptive to what it actually does (what type of build is this for? may be confused with backend-frontend-plugin builds)

Reply:
Okay! I will do that. makes sense!